CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,23 @@
+# CODE OF CONDUCT
+
+This code of conduct outlines our expectations for participants within the GirlScript Foundation [Link](https://www.gssoc.tech), as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```

Comment:
WrippleFOSS instead of girlscript

website link be https://wripplefoss.github.io/